README.md

```diff
@@ -1,3 +1,29 @@
 # fabric8-gemini-server
 Fabric8-server analytics powered services to initiate and report the readiness of
 all registered services
+
+### ENDPOINTS
+
+#### POST: /api/v1/register
```

Comment:
so the all the utility APIs are pre-pended with `/api/v1` string?

ping @samuzzal-choudhury , as I need to update scanner side of URLs to match.

Reply:
@navidshaikh, that's a standard structure I have observed in all other projects of fabric8. I guess we can go with pre pended string `/api/v1`

Reply:
@navidshaikh `/api/v1` is the way for us to manage any subsequent versions of the API. Currently, it's `api/v1`. Please make the necessary changes on the scanner side.